render-items.js

```diff
@@ -7,8 +7,12 @@ const renderProducts = (product) => {
     header3.textContent = product.name;
     li.appendChild(header3);
 
-    const USD = document.createElement('p');
-    USD.textContent = '$' + product.price.toFixed(2);
+    const p = document.createElement('p');
+    p.className = product.price;
```

Comment:
Do we want to give our paragraph a class that equals the product's price (like 4.99) or do we want to give it the string 'price'?